src/components/BoxColor.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+function BoxColor(props) {
+
+    const{r,g,b} = props
+    const divStyle = {
+        width: "700px",
+        height: "100px",
+        backgroundColor: `rgb(${r}, ${g}, ${b})`,
+        border: "5px solid #282c34",
+        margin: "10px"
+    }
+
```

Comment:
If you wanted the bonus, you just needed a function that converts `rgb` to `hex values`.
```
function rgbToHex(r, g, b) {
    return '#' + ((1 << 24) | (r << 16) | (g << 8) | b).toString(16).slice(1);
  }
```
It's not something you really need to know, but with a small search you could get this coverter.